Sources/AngouriMath/Core/Parser.cs

```diff
@@ -32,7 +34,8 @@ internal sealed class AngouriMathTextWriter : TextWriter
         }
         public static Entity Parse(string source)
         {
-            var lexer = new AngouriMathLexer(new AntlrInputStream(source), null, new AngouriMathTextWriter());
+            AngouriMathTextWriter angouriMathTextWriter = new AngouriMathTextWriter();
```

Comment:
Might make sense to replace this long type with just `var` 😄

Reply:
Or go for the `new()`.